realtime/utils.js

```diff
@@ -389,20 +393,16 @@ function attachAspectSubject(sample, subjectModel, aspectModel) {
   const aspName = nameParts[1];
   let promiseArr = [];
   if (featureToggles.isFeatureEnabled('attachSubAspFromDB')) {
-    const subOpts = {
-      where: {
-        absolutePath: { $iLike: subName },
-      },
-    };
-    const aspOpts = {
-      where: {
-        name: { $iLike: aspName },
-      },
-    };
     const getAspectPromise = sample.aspect ? Promise.resolve(sample.aspect) :
-      aspectModel.findOne(aspOpts);
+    sequelize.query('SELECT * FROM "Aspects" WHERE id = :aspectId and ' +
```

Comment:
`SELECT *` -- is the real-time sample event going to contain the same list of aspect attributes as before?

Reply:
I did some spot checking in sandbox and prod. The aspect attributes are the same in both sandbox and prod.